src/components/functionselect/functionselect.js

```diff
@@ -62,24 +65,11 @@ angular.module('vlui')
             'variance', 'variancep'
           ] // hide COUNT for Q in the UI because we dedicate it to a special "# Count" field
         };
-        aggregates.all = aggregates.aboveFold.concat(aggregates.belowFold)
-          .concat([COUNT]); // COUNT is a valid aggregate
+        quantitativeFunctions.all = _.concat(quantitativeFunctions.aboveFold, quantitativeFunctions.belowFold);
 
-        function getTimeUnits(type) {
-          if (type === 'temporal') {
-            return timeUnits.all;
-          }
-          return [];
-        }
-
-        function getAggregates(type) {
-          // HACK
-          // TODO: make this correct for temporal as well
-          if (type === 'quantitative' ){
-            return aggregates.all;
-          }
-          return [];
-        }
+        // aggregates = Q Functions + COUNT - BIN - undefined
+        var aggregates = _.pull(_.concat(quantitativeFunctions.all, [COUNT]), 
```

Comment:
From `pull`'s documentation

> Unlike `_.without`, this method mutates array

I don't understand why you want to mutate temporalFunctions.all.  

In general, you should avoid causing side-effect unless you have convincing reason to do so.